package.json

```diff
@@ -13,6 +13,7 @@
     "scripts": {
         "start": "node index.js",
         "dev": "nodemon index.js",
+        "update-theme": "node ./scripts/update-theme.mjs",
```

Comment:
```suggestion
```

Script no longer required (now, revering back to importing via npm package dependency)